src/drivers/rover_interface/scout_sdk/include/scout_sdk/ScoutRobot.hpp

```diff
@@ -1,5 +1,4 @@
-#ifndef SCOUT_ROBOT_HPP
```

Comment:
Maybe squash these header file changes to the first commit, where the files are added?